content/en/agent/proxy.md

```diff
@@ -293,113 +309,368 @@ backend datadog-metrics
     balance roundrobin
     mode http
     # The following configuration is for HAProxy 1.8 and newer
-    server-template mothership 5 haproxy-app.agent.{{< region-param key="dd_site" >}}:443 check port 443 ssl verify none check resolvers my-dns init-addr none resolve-prefer ipv4
+    server-template mothership 5 haproxy-app.agent.{{< region-param key="dd_site" >}}:443 check port 443 ssl verify <PATH_TO_CERTIFICATES> check resolvers my-dns init-addr none resolve-prefer ipv4
```

Comment:
Do you think that keeping `ssl verify none` in the configuration and add explanation on how to verify Datadog certificate would be better ?

> This may be something to cover with a separate config type as well.

Do you mean with new tabs ?

Reply:
> Do you think that keeping ssl verify none in the configuration and add explanation on how to verify Datadog certificate would be better ?

To me that would be the least friction for adopters but I can't comment on it confidently since I don't know for sure what the specific intents of the effort are.

> Do you mean with new tabs ?

Just mean that we can have a config with `ssl verify none` and then one with either an explanation (like you mentioned) or a full new config that has verification of the backend.

Reply:
Honestly I don't really know what to think about this part. I feel like we should prioritize 'security' and provide 'correct' guidelines (actual ones have security issues) even if it increases the setup / maintenance complexity.

But I completely understand the fact that we must keep the configuration simple. Imo the current trade-off is : 

## With the certificate verification 
- Pros :
     - More accurate guidelines / more secure
     - More awareness from customers about certificates verification (as they need to setup it)
- Cons :
     - More complexity on the setup / maintenance 

## Without verification 
- Pros : 
    -  Easy to setup
- Cons : 
    - Not optimal guidelines 
    -  Customers might not be aware of the provided configuration problems

Did I miss something ? 

My opinion is that I would let the configuration as it and add more details about how to do deal with `<PATH_TO_CERIFICATES>`  (they can still replace it by `none` as an alternative). But you have a better vision than me about what is better for customers and how you want the proxy guidelines to evolve in the future so feel free to decide.